docs/sections/user_guide/yaml/components/shave.rst

```diff
@@ -27,7 +27,7 @@ Describes the required parameters to run a ``shave`` configuration.
 
   Name of the grid file with extra points to be shaved.
 
-  **nh4:**
+  **nhalo:**
```

Comment:
`nh4` is a SRW construct to indicate specifically that there are 4 halo cells. Changed this name to reflect that it is the number of halo cells.